crates/bevy_hierarchy/src/child_builder.rs

```diff
@@ -195,6 +225,23 @@ impl<'w, 's, 'a> BuildChildren for EntityCommands<'w, 's, 'a> {
         self
     }
 
+    fn add_children<T>(&mut self, spawn_children: impl FnOnce(&mut ChildBuilder) -> T) -> T {
```

Comment:
`with_children` should be implemented in terms of this:

```rust
    self.add_children(spawn_children);
    self
```
(I think this should work)

Reply:
Yup, looks like it.